src/internal/pjsdb/job_crud.go

```diff
@@ -197,6 +212,48 @@ func WalkJob(ctx context.Context, tx *pachsql.Tx, id JobID) ([]Job, error) {
 	return jobs, nil
 }
 
+func walkLevelOrder(ctx context.Context, tx *pachsql.Tx, id JobID) (records []jobRecord, err error) {
+	records = make([]jobRecord, 0)
```

Comment:
`var records []jobRecord` is preferred over this declaration syntax: https://go.dev/wiki/CodeReviewComments#declaring-empty-slices

Reply:
I think I did this because I had wrongly assumed that SelectContext required the array to be already initialized. However I must have hallucinated that.

Reply:
These two forms (`var x []T` and `x := make([]T, 0)`) are basically equivalent.  (Slice with len = 0, cap = 0.)  If you were passing this to (io.Writer).Read, then it's wanting you to set the length of the read by setting the length of the slice and `make([]byte, 42)` would be appropriate for that.

Reply:
Oh and when it's the return value like this you don't even need `var`.  It's ready to append to.